Data-Analytics/Spark-GPU/Python/scala-gpu-example.yaml

```diff
@@ -14,15 +14,15 @@ spec:
     spark.task.resource.gpu.amount: "1"
     spark.executor.resource.gpu.amount: "1"
     spark.executor.resource.gpu.vendor: "nvidia.com"
-    spark.executor.resource.gpu.discoveryScript: "/opt/mapr/spark/spark-3.4.1/examples/src/main/scripts/getGpusResources.sh"
-    spark.rapids.shims-provider-override: "com.nvidia.spark.rapids.shims.spark341.SparkShimServiceProvider"
+    spark.executor.resource.gpu.discoveryScript: "/opt/mapr/spark/spark-3.5.0/examples/src/main/scripts/getGpusResources.sh"
+    spark.rapids.shims-provider-override: "com.nvidia.spark.rapids.shims.spark350.SparkShimServiceProvider"
  
   type: Python
-  sparkVersion: 3.4.1
+  sparkVersion: 3.5.0
   mode: cluster
-  image: gcr.io/mapr-252711/spark-gpu-3.4.1:v3.4.1
+  image: gcr.io/mapr-252711/spark-gpu-3.5.0:v3.5.0
   imagePullPolicy: Always
-  mainApplicationFile: .../path/to/application.py
+  mainApplicationFile: "local:///ezua-tutorials/Data-Analytics/Spark-GPU/Python/SqlOnGpuExample.py"
```

Comment:
Shouldn't we cover how to make this file available inside the running application?
Or we leave this to figure out to user?

Reply:
We should mount shared volume and then read it from shared volume. Also, image with ezua-tutorials should be rebuilt and updated in respective files.